server/game/GameActions/DeckSearchAction.ts

```diff
@@ -35,6 +36,13 @@ export class DeckSearchAction extends PlayerAction {
         reveal: true
     };
 
```

Comment:
Not really a fan of using underscore js when you are in a TS file and could type the argument as `amount: () => number || number`. And then do some type checking the TS way.